packages/frontend/src/pages/admin/index.vue

```diff
@@ -115,6 +115,11 @@ const menuDef = $computed(() => [{
 		text: i18n.ts.customEmojis,
 		to: '/admin/emojis',
 		active: currentPage?.route.name === 'emojis',
+	}, {
+		icon: 'ti ti-sparkles',
+		text: i18n.ts.avatarDecorations,
+		to: '/avatar-decorations',
```

Comment:
コントロールパネルから「アイコンデコレーション」を見に行ったときにサイドバーが消えてしまうのを修正したほうが良さそうです
```suggestion
		to: '/admin/avatar-decorations',
```